models/src/it/java/io/sphere/sdk/reviews/ReviewIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+                            optReview = result.getResults().stream().filter(review1 -> review1.getId().equals(review.getId())).findFirst();
+                            assertTrue(optReview.isPresent());
+                            assertEquals(optReview.get(), review);
+                        })));
+    }
+
+    @Test
+    public void testUpdateAuthorName() throws Exception {
+        withProduct(client(), product ->
+                withCustomer(client(), customer ->
+                        withReview(client(), product, customer, newReview -> {
+                            Instant startUpdate = Instant.now().minusSeconds(SERVER_TIME_TOLERANCE_SEC); // time tolerance
+                            Review review = client().execute(ReviewUpdateCommand.of(
+                                    Versioned.of(newReview.getId(), newReview.getVersion()), SetAuthorName.of(Optional.of((String) REVIEWS[1][0]))));
+                            Instant afterUpdate = Instant.now().plusSeconds(SERVER_TIME_TOLERANCE_SEC); // seconds tolerance
+                            assertEquals((String) REVIEWS[1][0], review.getAuthorName().get());
+                            assertNotEquals(newReview.getVersion(), review.getVersion());
+                            checkTime(review.getLastModifiedAt(), review.getLastModifiedAt(),
+                                    startUpdate, afterUpdate);
+                            review = client().execute(ReviewUpdateCommand.of(
+                                    Versioned.of(review.getId(), review.getVersion()), SetAuthorName.of(Optional.empty())));
+                            assertFalse(review.getAuthorName().isPresent());
+                        })));
+    }
+
+    @Test
+    public void testUpdateTitle() throws Exception {
+        withProduct(client(), product ->
+                withCustomer(client(), customer ->
+                        withReview(client(), product, customer, newReview -> {
+                            Instant startUpdate = Instant.now().minusSeconds(SERVER_TIME_TOLERANCE_SEC); // time tolerance
+                            Review review = client().execute(ReviewUpdateCommand.of(
+                                    Versioned.of(newReview.getId(), newReview.getVersion()), SetTitle.of(Optional.of((String) REVIEWS[1][1]))));
+                            Instant afterUpdate = Instant.now().plusSeconds(SERVER_TIME_TOLERANCE_SEC); // seconds tolerance
+                            assertEquals((String) REVIEWS[1][1], review.getTitle().get());
+                            assertNotEquals(newReview.getVersion(), review.getVersion());
+                            checkTime(review.getLastModifiedAt(), review.getLastModifiedAt(),
+                                    startUpdate, afterUpdate);
+                            review = client().execute(ReviewUpdateCommand.of(
+                                    Versioned.of(review.getId(), review.getVersion()), SetTitle.of(Optional.empty())));
```

Comment:
Instead of `Versioned.of(review.getId(), review.getVersion())` you can use directly `review`.